### ISSUE TITLE
Clicking Pause and then Play has a typo

### ISSUE THREAD

**Issue by WillAutioItrax** [jamesmontemagno/Hanselman.Forms#57]
Issue #17 touched on this. It seems there is a typo in OnPlayClicked().
This works for me:

>         private async void OnPlayClicked(object sender, EventArgs e)
        {
            //await CrossMediaManager.Current.VideoPlayer.Play();
            await CrossMediaManager.Current.Play();
            pause.IsEnabled = true;
            stop.IsEnabled = true;
        }

**Comment by jamesmontemagno** [comment]
Send a PR down :)

**Comment by WillAutioItrax** [comment]
Something is not working. I created a new VS project from GitHub. Created a Branch. Made the change. Tested it. Saved it locally. Tried to Push the Branch. That failed because:

> Pushing Fix-For_Issue_57
Error encountered while pushing branch to the remote repository: Git failed with a fatal error.
unable to access 'https://github.com/jamesmontemagno/Hanselman.Forms.git/': The requested URL returned error: 403
Pushing to https://github.com/jamesmontemagno/Hanselman.Forms.git

I asked my friend Google and tried a few of their suggestions to no avail. Any ideas on how to get past this? Thanks!
